configs/ci/roles/ros.json

```diff
@@ -0,0 +1,19 @@
+{
+    "roles": [
+        {
+          "name": "ROS Administrator",
+          "description": "Perform any available operation against any Resource Optimization service.",
+          "system": true,
+          "platform_default": true,
+          "version": 8,
```

Comment:
It doesn't really matter, but just curious if this is version 8 for a reason?

Reply:
It should be 1, right? Since the first version of ROS  rbac config.

Reply:
@coderbydesign, 
Here, I would like to understand meaning of version.
Is it something similar to how database schema version works? I mean in this case for particular role object. Each time, one needs to update version value to reflect new changes to particular role object, right?

Reply:
Since these config files are used to seed data for all tenants, whenever there's a change made to the config, you also need to bump the version number in order for our seeding logic to know there is a change which should be applied.

Reply:
Thank you @coderbydesign for details. 
 
@apuntamb - could you please set version value to 1?